File: articles/probabilistic-algorithms-and-reasoning-under-uncertainty/typescript/probabilityCalculator.ts
function standardError(pHat: number, n: number): number {
  return Math.sqrt((pHat * (1 - pHat)) / n);
}

const pHat = 0.57;
const n = 1000;
console.log(JSON.stringify({ pHat, n, standardError: standardError(pHat, n) }, null, 2));

File: articles/probabilistic-algorithms-and-reasoning-under-uncertainty/fortran/probability_calculator.f90
program probability_calculator
  implicit none
  real :: p_hat, n, se
  p_hat = 0.57
  n = 1000.0
  se = sqrt((p_hat * (1.0 - p_hat)) / n)
  print *, "p_hat=", p_hat, " standard_error=", se
end program probability_calculator
